command-expansion-server/src/packages/lib/CommandTypeCodegen.ts

```diff
@@ -199,7 +199,7 @@ function versionBuilder(dictionary: ampcs.CommandDictionary) {
 }
 
 export async function processDictionary(dictionary: ampcs.CommandDictionary) {
-  const prefaceUrl = new URL("./CommandEDSLPreface.ts", import.meta.url);
+  const prefaceUrl = new URL("./packages/lib/CommandEDSLPreface.js", import.meta.url);
```

Comment:
Hmmm....interesting problem I didn't think about...we want the typescript version of this for type checking but the output format that would be relative to the file being executed at runtime would be a .js file ...hmmm

Reply:
Might need to have a build step to copy over the .ts file for that purpose. That can be handled later in the execution PR though.

Reply:
Added a copy command to the `npm run build`